pkg/networkmanager/interfaces.js

```diff
@@ -34,7 +34,7 @@ const _ = cockpit.gettext;
 function show_error_dialog(title, message) {
     const props = {
         id: "error-popup",
-        title: title,
+        title,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)